ansible/roles/ocp-client-vm/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+non_root_os_user: "ec2-user"
+maven_version: "3.5.4"
+docker_device: /dev/xvdb
```

Comment:
@jbride  Where is this docker_device being used? I don't see this anywhere in your changes.